ocr/tess/crop_morphology.py

```diff
@@ -76,10 +73,24 @@ def crop_area(crop):
 def find_border_components(contours, ary):
     borders = []
     area = ary.shape[0] * ary.shape[1]
+    print(f'contours: {len(contours)}')
+    # debug = np.zeros(ary.shape, dtype=np.uint8)
     for i, c in enumerate(contours):
         x,y,w,h = cv2.boundingRect(c)
-        if w * h > 0.5 * area:
+        rot_rect = cv2.minAreaRect(c)
+        ((rx, ry), (rw, rh), deg) = rot_rect
+        if 5 < deg < 45 and rw * rh > 1000:
+            print(f'{i}: {rx:.0f},{ry:.0f} {rw:.0f}x{rh:.0f} +{deg}°')
+            # box = cv2.boxPoints(rot_rect)
+            # box = np.int0(box)
+            # cv2.drawContours(debug, [box], 0, 255, 2)
+        if w * h > 0.5 * area and angle_from_right(deg) < 5:
```

Comment:
If this detected a heavily-rotated rectangle, chaos could ensue. Example: 717896b.